lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -39,7 +39,19 @@ use virt_utils;
 
 our @EXPORT
   = qw(download_network_cfg prepare_network restore_standalone destroy_standalone restart_libvirtd restart_network restore_guests restore_network
-  destroy_vir_network restore_libvirt_default enable_libvirt_log ssh_setup upload_debug_log check_guest_status save_guest_ip test_network_interface hosts_backup hosts_restore);
+  destroy_vir_network restore_libvirt_default enable_libvirt_log ssh_setup upload_debug_log check_guest_status check_guest_module save_guest_ip test_network_interface hosts_backup hosts_restore);
+
+sub check_guest_module {
+    my ($guest, %args) = @_;
+    my $module = $args{module};
+    if (($guest =~ m/sles-?11/i) && ($module eq "acpiphp")) {
+        my $status = script_run("ssh root\@$guest \"lsmod | grep $module\"");
+        if ($status != 0) {
+            script_run("ssh root\@$guest modprobe $module", 60);
+            record_soft_failure 'bsc#1167828 - need to load acpiphp kernel module to sles11sp4 guest otherwise network interface hotplugging does not work';
```

Comment:
@guoxuguang Please be aware that softfailure is for those workarounds for valid bugs which are confirmed and will be fixed by developers. But for this bug bsc#1167828, dev won't fix it, and it is closed. So it is not softfail, it is JUST THE SLE11SP4 SPECIAL PREPARATION needed.
So please remove this record_soft_failure, if you want to record some info, suggest to use record_info only.

Reply:
Had removed this record_soft_failure, and used with record_info only already.

Reply:
if there was not any comments, please help me merge this PR.